dev/bin/manager-run.sh

```diff
@@ -58,11 +58,6 @@ mvn \
   -Dmanaged-connectors.auth.offline-token=${MANAGED_CONNECTORS_AUTH_OFFLINE_TOKEN} \
   \
   -Dquarkus.devservices.enabled=false \
-  -Dkafka.client.id=${mc_client_id} \
```

Comment:
@robpblake yes these were placeholder parameters that I put here when the manager was importing the kafka client only because of messy dependency tree, to avoid printing a warning message every few seconds when I run the manager locally.

Now that the manager imports a kafka client because it *really* needs it, it's time to remove them.

Reply:
Thanks.